File: examples-sdk/typescript/703-protocol-soap-client.ts
#!/usr/bin/env tsx
// 703-protocol-soap-client — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 703-protocol-soap-client.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();

File: examples-sdk/python/703-protocol-soap-client.py
#!/usr/bin/env python3
"""703-protocol-soap-client — Python SDK equivalent
Compile: vil compile --from python --input 703-protocol-soap-client.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()
